src/Umbraco.Tests.UnitTests/Umbraco.Core/Models/PathValidationTests.cs

```diff
@@ -7,7 +7,7 @@
 using Umbraco.Tests.Common.Builders;
 using Umbraco.Tests.Common.Builders.Extensions;
 
-namespace Umbraco.Tests.UnitTests.Umbraco.Infrastructure.Models
+namespace Umbraco.Tests.UnitTests.Umbraco.Core.Models
```

Comment:
Same case as with **DataTypeTests.cs**

-  `PathValidationExtensions` class 
   - `Umbraco.Core` namespace
   - located in _Umbraco.Infrastructure_ folder
   - _Microsoft.Extensions.Logging_ using

Reply:
Also moved to the right place.